Scheduler: update code snippets (#4475)
DevExpress/devextreme-documentation: concepts/05 UI Components/Scheduler/040 Resources/040 Customize Resource Headers.md

---

##### Angular

>     For an Angular app, DevExtreme includes the [dxTemplate](/api-reference/10%20UI%20Components/Markup%20Components/dxTemplate '/Documentation/ApiReference/UI_Components/Markup_Components/dxTemplate/') markup component. The following code shows how to use dxTemplate to define custom templates for resource headers.

Reviewer: ```suggestion
    For an Angular apps, DevExtreme comes with the [dxTemplate](/api-reference/10%20UI%20Components/Markup%20Components/dxTemplate '/Documentation/ApiReference/UI_Components/Markup_Components/dxTemplate/') markup component. The following code shows how to use dxTemplate to define custom templates for resource headers.
```

##### Vue

> Templates allow you to customize UI components. DevExtreme templates make use of the functionality provided by Vue. Implement a custom template and assign it to the [resourceCellTemplate](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/#resourceCellTemplate) property as shown in the following code.

Reviewer: There is something strange about your second sentence here. Can you rewrite it so that there is a better flow of information and a clearer relationship between the first sentence and the second. You can probably just combine these into one sentence.

Reviewer 2: DevExtreme templates allow you to customize UI components and make use of the functionality provided by Vue. Implement a custom template and assign ...

##### React

>     Implement a callback function with custom template and assign it to the [resourceCellRender](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/#resourceCellRender) property.

Reviewer: ```suggestion
    Implement a callback function with a custom template and assign it to the [resourceCellRender](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/#resourceCellRender) property.
```

>     Templates allow you to customize UI components. Implement a callback function with custom template and assign it to the [resourceCellRender](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/#resourceCellRender) property as shown in the following code.

Reviewer: ```suggestion
    Templates allow you to customize UI components. Implement a callback function with custom template and assign it to the [resourceCellRender](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/#resourceCellRender) property as shown in the following code.
```
"Implement a callback function with custom template " this part doesn't make sense; is there something missing here? can you rewrite it? the relationship of the callback function and the custom template is not clear.

Reviewer 2: DevExtreme templates allow you to customize UI components. Implement a custom template and assign it to the **resourceCellRender** property as shown in the following code.
